cime_config/config_component.xml

```diff
@@ -103,10 +103,11 @@
 
   <entry id="CAM_DYCORE">
     <type>char</type>
-    <valid_values>eul,fv,se,none</valid_values>
+    <valid_values>eul,fv,mpasa,se,none</valid_values>
```

Comment:
If you use the CIME Makefile method then I believe the `CAM_DYCORE` variable should be set to `mpas` instead of  `mpasa`.

Reply:
Since I switched back to use CIME Makefile for building, this change has been reverted in https://github.com/ESCOMP/CAM-SIMA/pull/245/commits/38d0de2f3891a56c4d1a8d00d2f9234fc91f692a.